source/agora/node/Network.d

```diff
@@ -78,12 +81,16 @@ public class NetworkManager
         // the node communicating with itself
         this.banned_addresses.put(
             format("http://%s:%s", this.node_config.address, this.node_config.port));
+
+        this.dns_seeds = dns_seeds;
     }
 
     /// try to discover the network until we found
     /// all the validator nodes from our quorum set.
     public void discover ()
     {
+        logInfo("Resolving DNS from %s", this.dns_seeds);
+        this.addAddresses(resolveDnsSeeds(this.dns_seeds));
```

Comment:
Also what that in a `if (this.dns_seeds.length)`
We don't want to make DNS seed mandatory for the moment, do we ?